### PR TITLE
fix: avoids negative values on inputs (fixes #456) and fixes linter w

### LINKED ISSUE #456: Validate negative Price/GB and Size field for Offers
When creating an Offer with Billing plans it nows allows you to add negative price. And send the transaction.

We should add the validation to avoid negative values, only include values >0.

image

Same happens for SIZE:

image

### PR DESCRIPTION
**Pull request by farinalvaro**
- Fixes #456 
- Cleans some linter warnings

### CODE REVIEW

**Review by julianmrodri — DISMISSED** [review]
LGTM

File: src/components/organisms/storage/sell/EditOfferStepper.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const EditOfferStepper: FC<EditOfferStepperProps> = ({ endHandler }) => {
               onClick={handleNext}
               className={classes.button}
               rounded
-              disabled={!availableSize || !system || !peerId}
+              disabled={!availableSize || availableSize <= 0 || !system || !peerId}
```
Comment (jurajpiar) [inline src/components/organisms/storage/sell/EditOfferStepper.tsx:54]: might as well extract that to a const with this size

File: src/components/organisms/storage/sell/EditableBillingPlan.tsx [hunk 2]
```diff
@@ -72,25 +83,30 @@ const EditableBillingPlan: FC<EditableBillingPlanProps> = ({
...
       return (
         <Button
           color="primary"
           variant="outlined"
           rounded
           onClick={handleOnAddClick}
-          disabled={Number(pricePerGb) <= 0 || usedPeriodsPerCurrency[currency]?.includes(period)}
+          disabled={
+            Number(pricePerGb) <= 0
+            || usedPeriodsPerCurrency[currency]?.includes(period)
```
Comment (jurajpiar) [inline src/components/organisms/storage/sell/EditableBillingPlan.tsx:96]: extract const

**Review by jurajpiar — APPROVED** [review]
Just suggested a couple of things but LG

**Review by julianmrodri — APPROVED** [review]
LGTM

**Review by jurajpiar — APPROVED** [review]

**Pull request merged**